charts/neon-storage-broker/README.md

```diff
@@ -64,4 +64,4 @@ Kubernetes: `^1.18.x-x`
 | tolerations | list | `[]` | Tolerations for pod assignment. |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.9.1](https://github.com/norwoodj/helm-docs/releases/v1.9.1)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
This one causes the `ct` to detect the chart has changed

Reply:
I see! Thanks!